geomstats/geometry/special_orthogonal.py

```diff
@@ -305,13 +306,27 @@ class _SpecialOrthogonalVectors(LieGroup):
 
     def __init__(self, n, epsilon=0.0, equip=True):
         dim = n * (n - 1) // 2
-        super().__init__(dim=dim, shape=(dim,), lie_algebra=self, equip=equip)
+
+        super().__init__(
+            dim=dim, shape=(dim,), lie_algebra=Euclidean(dim, equip=False), equip=equip
```

Comment:
from a design perspective, `lie_algebra = Euclidean` feels a bit weird, because Euclidean doesn't have a lie bracket, thus how could it be a lie algebra?